docs/governance/templates/release_issue.md

```diff
@@ -44,9 +44,9 @@ and copy it into a release issue. Fill in relevant values, found inside {}
 - [ ] Submit the Release.
 - [ ] Send an email to the [mailing list][list] with the release details (copy-paste the release blog post)
 - [ ] If full release, then increment the `base_version` in [`build/Makefile`][build-makefile]
-- [ ] If full release move [helm `tag` value][values] is set to {version}+1
-- [ ] If full release move the [helm `Chart` version values][chart] is to {version}+1
-- [ ] If full release move the [`sdks/nodejs/package.json`][nodejs] to {version}+1
+- [ ] If full release move [helm `tag` value][values] is set to {version}+1-dev
```

Comment:
@roberthbailey - I put in place your suggestions for naming of dev builds. I assume this is what you were referring to?

Reply:
yep.